faunadb/query.py

```diff
@@ -9,10 +9,13 @@
 
 # pylint: disable=invalid-name, redefined-builtin
 
+from collections import Mapping, namedtuple, Sequence
```

Comment:
`from collections import Mapping, Sequence, namedtuple` (I know only the most important of issues)